sdktests/server_side_migrations.go

```diff
@@ -50,7 +50,7 @@ func runMigrationVariationTests(t *ldtest.T) {
 			IsSummaryEvent(),
 		))
 
-		assert.Equal(t, stage, response.Result)
+		assert.Equal(t, string(stage), response.Result)
```

Comment:
There's some assertion that will do this for you. I think maybe.. `assert.EqualValues`?